mcproto/entity.py

```diff
@@ -4,7 +4,7 @@
 from functools import partial
 
 from ._base import HasStub, _EntityProvider
-from ._types import COLOR
+from ._types import COLOR, EFFECT
```

Comment:
Konnte wegen Typo dieses Befehl nicht testen (in _types heißt der Typ EFFECT*E*, bitte dort ausbesser), im Moment ist das ein ImportError!